packages/components/src/components/step-group/step-group.stories.ts

```diff
@@ -44,240 +53,184 @@ export default {
- * The size of the icon is recommended to be 48px for the 'lg' size and 32px for the 'sm' size. The size can be set using font-size.
+ * Use the `active-step` attribute to set the current step in the step-group.
  */
 
-export const SampleNotInteractive = {
-  parameters: { controls: { include: [] } },
-  name: 'Sample: Not Interactive',
-  render: () => {
-    return html`
-      <style>
-        .headline {
-          padding: 16px;
-          background: #e0e0e0;
-          text-align: left;
-          font-size: 14px;
-          font-weight: bold;
-          width: 100%;
-          box-sizing: border-box;
-        }
-      </style>
-      <div class="headline">size = 'lg'</div>
-      <sd-step-group size="lg" orientation="horizontal" not-interactive>
-        <sd-step orientation="horizontal">
-          <p slot="label">Lorem ipsum dolor sit</p>
-          <sd-icon class="text-[48px]" name="content/image" slot="circle-content"></sd-icon>
+export const ActiveStep = {
+  name: 'Active Step',
+  render: () => html`
+    <div class="space-y-5">
+      <sd-step-group size="lg" orientation="horizontal" active-step="0">
+        <sd-step size="lg" orientation="horizontal" state="default">
+          <p slot="label">Label</p>
         </sd-step>
 
-        <sd-step orientation="horizontal">
-          <p slot="label">Exercitation ullamco laboris</p>
-          <sd-icon class="text-[48px]" name="content/image" slot="circle-content"></sd-icon>
+        <sd-step size="lg" orientation="horizontal" state="current">
+          <p slot="label">Label</p>
         </sd-step>
 
-        <sd-step orientation="horizontal">
-          <p slot="label">Reprehenderit qui in e name</p>
-          <sd-icon class="text-[48px]" name="content/image" slot="circle-content"></sd-icon>
+        <sd-step size="lg" orientation="horizontal" state="disabled">
+          <p slot="label">Label</p>
         </sd-step>
       </sd-step-group>
-      <div class="headline">size = 'sm'</div>
-      <sd-step-group size="sm" orientation="horizontal" not-interactive>
-        <sd-step orientation="horizontal">
-          <p slot="label">Lorem ipsum dolor sit</p>
-          <sd-icon class="text-3xl" name="content/image" slot="circle-content"></sd-icon>
+
+      <div class="flex gap-12 h-[20vh]">
+        <sd-step-group size="lg" orientation="vertical" active-step="1">
+          <sd-step size="lg" orientation="vertical" state="default">
+            <p slot="label">Label</p>
+          </sd-step>
+
+          <sd-step size="lg" orientation="vertical" state="current">
+            <p slot="label">Label</p>
+          </sd-step>
+
+          <sd-step size="lg" orientation="vertical" state="disabled">
+            <p slot="label">Label</p>
+          </sd-step>
+        </sd-step-group>
+      </div>
+    </div>
+  `
+};
+
+/**
+ * Use the `not-interactive` attribute to create a non-interactive step group.
+ */
+
+export const notInteractive = {
+  name: 'Not Interactive',
```

Comment:
We already have an issue to fix the center alignment of circle and label below. Right, @Vahid1919 ?